src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const myList = document.querySelector('.tree');
+
+const listItems = myList.getElementsByTagName('li');
+
+for (let i = 0; i < listItems.length; i++) {
+  listItems[i].hidden = false;
+
+  const newSpan = document.createElement('span');
+
+  listItems[i].insertAdjacentElement('beforebegin', newSpan);
+  newSpan.append(listItems[i]);
+}
+
+const add = (e) => {
```

Comment:
Rename the function `add` to a more descriptive name like `toggleVisibility` to improve readability.